build-system/tasks/check-links.js

```diff
@@ -134,9 +135,15 @@ function checkLinksInFile(file) {
       }
       let containsDeadLinks = false;
       for (const {link, status, statusCode} of results) {
-        // Skip links to files that were introduced by the PR.
+        // Catch all files that were introduced by the PR.
         if (isLinkToFileIntroducedByPR(link)) {
```

Comment:
What if the link is a relative path?
```suggestion
        if (isLinkToFileIntroducedByPR(link) && status == 'dead') {
```